src/compounds/product-table/src/components/cell-content.tsx

```diff
@@ -0,0 +1,79 @@
+/** @jsx jsx */
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+
+import { AddonContext, CellContext } from '../generics'
+
+import CellBase from './cell-base'
+
+const grid = (rowOrCol: 'row' | 'column', start: number, span: number) => {
+  const rowOrColCap = rowOrCol[0].toUpperCase() + rowOrCol.slice(1)
```

Comment:
Though strings have array like properties and numeric operators, I wouldn't recommend uppercasing the first letting in this way - it's only a suggestion but it might be better to do:
```js
const capitalise = string => `${string.charAt(0).toUpperCase()}${string.slice(1)}`
...
const rowOrColCap = capitalise(rowOrCol)
````